.github/workflows/ci.yaml

```diff
@@ -90,8 +90,6 @@ jobs:
           SHORT_COMMIT=$(git rev-parse --short "$GITHUB_SHA")
           ./gradlew :core:python3Action:distDocker -PdockerRegistry=docker.io -PdockerImagePrefix=openwhisk -PdockerImageTag=nightly
           ./gradlew :core:python3Action:distDocker -PdockerRegistry=docker.io -PdockerImagePrefix=openwhisk -PdockerImageTag=$SHORT_COMMIT
-          ./gradlew :core:python36AiAction:distDocker -PdockerRegistry=docker.io -PdockerImagePrefix=openwhisk -PdockerImageTag=nightly
-          ./gradlew :core:python36AiAction:distDocker -PdockerRegistry=docker.io -PdockerImagePrefix=openwhisk -PdockerImageTag=$SHORT_COMMIT
```

Comment:
This will entirely remove AI runtime.
I am not sure we need to archive it for future reference(though we can still refer to the commit).

Reply:
Yeah, I hesitated on this which is why I didn't remove this back in early 2022 when we first hit the EOL date.  But at this point, Python 3.6 is more than a year past its EOL and all of the AI libraries included in this image are also obsolete.  So, I think this runtime is no longer useful to provide from the core project.   There is enough diversity in Python packages for AI, that its also not clear to me that we can hope to provide the "one true" combination of packages as a core runtime (as opposed to people building custom images and running as a blackbox or as a python virtualenv).

Reply:
Fair point.